exported_data_bucket_policy.tf

```
@@ -0,0 +1,21 @@
+resource "aws_s3_bucket_policy" "exported_data" {
+  bucket = aws_s3_bucket.exported_data.id
+  policy = data.aws_iam_policy_document.exported_data_bucket.json
+}
+
+data "aws_iam_policy_document" "exported_data_bucket" {
+  statement {
+    principals {
+      type        = "AWS"
+      identifiers = ["${var.findings_data_bucket_access_role_arn}"]
```

Does this need the string interpolation or can we just do:

```suggestion
      identifiers = [var.findings_data_bucket_access_role_arn]
```

---

You are correct - good catch, thanks!